### PR TITLE
Browser support update

### PR DESCRIPTION
**Pull request by mhevery**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: browser-providers.conf.js [hunk 1]
```diff
@@ -27,18 +27,14 @@ var CIconfiguration = {
...
-  'Safari9': {unitTest: {target: 'BS', required: false}, e2e: {target: null, required: true}},
-  'Safari10': {unitTest: {target: 'BS', required: false}, e2e: {target: null, required: true}},
-  'iOS7': {unitTest: {target: 'BS', required: true}, e2e: {target: null, required: true}},
-  'iOS8': {unitTest: {target: 'BS', required: false}, e2e: {target: null, required: true}},
-  'iOS9': {unitTest: {target: 'BS', required: false}, e2e: {target: null, required: true}},
-  'iOS10': {unitTest: {target: 'SL', required: false}, e2e: {target: null, required: true}},
+  'Android8': {unitTest: {target: 'SL', required: true}, e2e: {target: null, required: true}},
+  'Android9': {unitTest: {target: 'SL', required: true}, e2e: {target: null, required: true}},
+  'Android10': {unitTest: {target: 'SL', required: true}, e2e: {target: null, required: true}},
+  'Safari12': {unitTest: {target: 'BS', required: false}, e2e: {target: null, required: true}},
```
Comment (IgorMinar) [inline browser-providers.conf.js]: We currently don't run any browser stack (`BS`) jobs on CI. Can we use `SL` instead?
```suggestion
  'Safari12': {unitTest: {target: 'SL', required: false}, e2e: {target: null, required: true}},
```

File: browser-providers.conf.js [hunk 2]
```diff
@@ -27,18 +27,14 @@ var CIconfiguration = {
...
+  'Safari13': {unitTest: {target: 'BS', required: false}, e2e: {target: null, required: true}},
```
Comment (IgorMinar) [inline browser-providers.conf.js]:
```suggestion
  'Safari13': {unitTest: {target: 'SL', required: false}, e2e: {target: null, required: true}},
```

File: browser-providers.conf.js [hunk 3]
```diff
@@ -27,18 +27,14 @@ var CIconfiguration = {
...
+  'iOS10': {unitTest: {target: 'BS', required: false}, e2e: {target: null, required: true}},
```
Comment (IgorMinar) [inline browser-providers.conf.js]:
```suggestion
  'iOS10': {unitTest: {target: 'SL', required: false}, e2e: {target: null, required: true}},
```

**Review by IgorMinar — CHANGES_REQUESTED** [review]
```
07 02 2020 00:34:25.948:ERROR [launcher]: Cannot load browser "SL_ANDROID8": it is not registered! Perhaps you are missing some plugin?
07 02 2020 00:34:25.948:ERROR [launcher]: Cannot load browser "SL_ANDROID9": it is not registered! Perhaps you are missing some plugin?
07 02 2020 00:34:25.949:ERROR [launcher]: Cannot load browser "SL_ANDROID10": it is not registered! Perhaps you are missing some plugin?
0
```

you need to define the browsers in `customLanchers` section below the diff you made.

File: browser-providers.conf.js (hunk 1 above)
Comment (mhevery) [reply browser-providers.conf.js]: fixed

File: browser-providers.conf.js (hunk 2 above)
Comment (mhevery) [reply browser-providers.conf.js]: fixed

File: browser-providers.conf.js (hunk 3 above)
Comment (mhevery) [reply browser-providers.conf.js]: fixed

**Review by gkalpak — APPROVED** [review]

**Review by IgorMinar — APPROVED** [review]
lgtm!

**Pull request closed without merging**